File: Java/app/src/main/java/com/example/randomizermenumakanan/Model/IsiMenu.java
package com.example.randomizermenumakanan.Model;

public class IsiMenu {
    private String nama;
    private String tag;
    private String bahan;
    private String langkah;
    private String restoran;

    public IsiMenu(String nama, String tag, String bahan, String langkah, String restoran){
        this.nama = nama;
        this.tag = tag;
        this.bahan = bahan;
        this.langkah = langkah;
        this.restoran = restoran;
    }

    public void setNama(String nama){
        this.nama = nama;
    }

    public void setTag(String tag){
        this.tag = tag;
    }

    public void setBahan(String bahan){
        this.bahan = bahan;
    }

    public void setLangkah(String langkah){
        this.langkah = langkah;
    }

    public void setRestoran(String restoran){
        this.restoran = restoran;
    }

    public String getNama(){
        return this.nama;
    }

    public String getTag(){
        return this.tag;
    }

    public String getBahan(){
        return this.bahan;
    }

    public String getLangkah(){
        return this.langkah;
    }

    public String getRestoran(){
        return this.restoran;
    }
}

File: Kotlin/app/src/main/java/com/example/randomizermenumakanan/Model/IsiMenu.kt
package com.example.randomizermenumakanan.Model

class IsiMenu {
    private lateinit var nama: String
    private lateinit var tag: String
    private lateinit var bahan: String
    private lateinit var langkah: String
    private lateinit var restoran: String

    constructor(nama: String, tag: String, bahan: String, langkah: String, restoran: String){
        this.nama = nama
        this.tag = tag
        this.bahan = bahan
        this.langkah = langkah
        this.restoran = restoran
    }
    fun setNama(nama: String){
        this.nama = nama
    }
    fun setTag(tag: String){
        this.tag = tag
    }
    fun setBahan(bahan: String){
        this.bahan = bahan
    }
    fun setLangkah(langkah: String){
        this.langkah = langkah
    }
    fun setRestoran(restoran: String){
        this.restoran = restoran
    }

    fun getNama(): String{
        return this.nama
    }

    fun getTag(): String{
        return this.tag
    }

    fun getBahan(): String{
        return this.bahan
    }
    fun getLangkah(): String{
        return this.langkah
    }
    fun getRestoran(): String{
        return this.restoran
    }
}